JCL_Test_SE90+FAILED: test_getModifiers_classTypes

https://ci.eclipse.org/openj9/job/PullRequest-Sanity-JDK10-linux_ppc-64_cmprssptrs_le-OpenJ9/3/

```
FAILED: test_getModifiers_classTypes
java.lang.AssertionError: One or more tests failed:
Incorrect modifiers for org.openj9.test.java.lang.Test_Class$7. Expected <8>, Actual <0>.

	at org.testng.Assert.fail(Assert.java:96) from jdk.internal.loader.ClassLoaders$AppClassLoader@ca632023(file:/home/jenkins/jenkins-agent/workspace/PullRequest-Sanity-JDK10-linux_ppc-64_cmprssptrs_le-OpenJ9/openj9/jvmtest/TestConfig/lib/testng.jar)
	at org.openj9.test.java.lang.Test_Class.test_getModifiers_classTypes(Test_Class.java:1156) from jdk.internal.loader.ClassLoaders$AppClassLoader@ca632023(file:/home/jenkins/jenkins-agent/workspace/PullRequest-Sanity-JDK10-linux_ppc-64_cmprssptrs_le-OpenJ9/openj9/jvmtest/Java8andUp/GeneralTest.jar)
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(java.base@10-internal/Native Method) from jrt:/java.base
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(java.base@10-internal/NativeMethodAccessorImpl.java:62) from jrt:/java.base
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(java.base@10-internal/DelegatingMethodAccessorImpl.java:43) from jrt:/java.base
	at java.lang.reflect.Method.invoke(java.base@10-internal/Method.java:564) from jrt:/java.base
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124) from jdk.internal.loader.ClassLoaders$AppClassLoader@ca632023(file:/home/jenkins/jenkins-agent/workspace/PullRequest-Sanity-JDK10-linux_ppc-64_cmprssptrs_le-OpenJ9/openj9/jvmtest/TestConfig/lib/testng.jar)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:580) from jdk.internal.loader.ClassLoaders$AppClassLoader@ca632023(file:/home/jenkins/jenkins-agent/workspace/PullRequest-Sanity-JDK10-linux_ppc-64_cmprssptrs_le-OpenJ9/openj9/jvmtest/TestConfig/lib/testng.jar)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716) from jdk.internal.loader.ClassLoaders$AppClassLoader@ca632023(file:/home/jenkins/jenkins-agent/workspace/PullRequest-Sanity-JDK10-linux_ppc-64_cmprssptrs_le-OpenJ9/openj9/jvmtest/TestConfig/lib/testng.jar)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988) from jdk.internal.loader.ClassLoaders$AppClassLoader@ca632023(file:/home/jenkins/jenkins-agent/workspace/PullRequest-Sanity-JDK10-linux_ppc-64_cmprssptrs_le-OpenJ9/openj9/jvmtest/TestConfig/lib/testng.jar)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125) from jdk.internal.loader.ClassLoaders$AppClassLoader@ca632023(file:/home/jenkins/jenkins-agent/workspace/PullRequest-Sanity-JDK10-linux_ppc-64_cmprssptrs_le-OpenJ9/openj9/jvmtest/TestConfig/lib/testng.jar)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109) from jdk.internal.loader.ClassLoaders$AppClassLoader@ca632023(file:/home/jenkins/jenkins-agent/workspace/PullRequest-Sanity-JDK10-linux_ppc-64_cmprssptrs_le-OpenJ9/openj9/jvmtest/TestConfig/lib/testng.jar)
	at org.testng.TestRunner.privateRun(TestRunner.java:648) from jdk.internal.loader.ClassLoaders$AppClassLoader@ca632023(file:/home/jenkins/jenkins-agent/workspace/PullRequest-Sanity-JDK10-linux_ppc-64_cmprssptrs_le-OpenJ9/openj9/jvmtest/TestConfig/lib/testng.jar)
	at org.testng.TestRunner.run(TestRunner.java:505) from jdk.internal.loader.ClassLoaders$AppClassLoader@ca632023(file:/home/jenkins/jenkins-agent/workspace/PullRequest-Sanity-JDK10-linux_ppc-64_cmprssptrs_le-OpenJ9/openj9/jvmtest/TestConfig/lib/testng.jar)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455) from jdk.internal.loader.ClassLoaders$AppClassLoader@ca632023(file:/home/jenkins/jenkins-agent/workspace/PullRequest-Sanity-JDK10-linux_ppc-64_cmprssptrs_le-OpenJ9/openj9/jvmtest/TestConfig/lib/testng.jar)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450) from jdk.internal.loader.ClassLoaders$AppClassLoader@ca632023(file:/home/jenkins/jenkins-agent/workspace/PullRequest-Sanity-JDK10-linux_ppc-64_cmprssptrs_le-OpenJ9/openj9/jvmtest/TestConfig/lib/testng.jar)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415) from jdk.internal.loader.ClassLoaders$AppClassLoader@ca632023(file:/home/jenkins/jenkins-agent/workspace/PullRequest-Sanity-JDK10-linux_ppc-64_cmprssptrs_le-OpenJ9/openj9/jvmtest/TestConfig/lib/testng.jar)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364) from jdk.internal.loader.ClassLoaders$AppClassLoader@ca632023(file:/home/jenkins/jenkins-agent/workspace/PullRequest-Sanity-JDK10-linux_ppc-64_cmprssptrs_le-OpenJ9/openj9/jvmtest/TestConfig/lib/testng.jar)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52) from jdk.internal.loader.ClassLoaders$AppClassLoader@ca632023(file:/home/jenkins/jenkins-agent/workspace/PullRequest-Sanity-JDK10-linux_ppc-64_cmprssptrs_le-OpenJ9/openj9/jvmtest/TestConfig/lib/testng.jar)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84) from jdk.internal.loader.ClassLoaders$AppClassLoader@ca632023(file:/home/jenkins/jenkins-agent/workspace/PullRequest-Sanity-JDK10-linux_ppc-64_cmprssptrs_le-OpenJ9/openj9/jvmtest/TestConfig/lib/testng.jar)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208) from jdk.internal.loader.ClassLoaders$AppClassLoader@ca632023(file:/home/jenkins/jenkins-agent/workspace/PullRequest-Sanity-JDK10-linux_ppc-64_cmprssptrs_le-OpenJ9/openj9/jvmtest/TestConfig/lib/testng.jar)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137) from jdk.internal.loader.ClassLoaders$AppClassLoader@ca632023(file:/home/jenkins/jenkins-agent/workspace/PullRequest-Sanity-JDK10-linux_ppc-64_cmprssptrs_le-OpenJ9/openj9/jvmtest/TestConfig/lib/testng.jar)
	at org.testng.TestNG.runSuites(TestNG.java:1049) from jdk.internal.loader.ClassLoaders$AppClassLoader@ca632023(file:/home/jenkins/jenkins-agent/workspace/PullRequest-Sanity-JDK10-linux_ppc-64_cmprssptrs_le-OpenJ9/openj9/jvmtest/TestConfig/lib/testng.jar)
	at org.testng.TestNG.run(TestNG.java:1017) from jdk.internal.loader.ClassLoaders$AppClassLoader@ca632023(file:/home/jenkins/jenkins-agent/workspace/PullRequest-Sanity-JDK10-linux_ppc-64_cmprssptrs_le-OpenJ9/openj9/jvmtest/TestConfig/lib/testng.jar)
	at org.testng.TestNG.privateMain(TestNG.java:1354) from jdk.internal.loader.ClassLoaders$AppClassLoader@ca632023(file:/home/jenkins/jenkins-agent/workspace/PullRequest-Sanity-JDK10-linux_ppc-64_cmprssptrs_le-OpenJ9/openj9/jvmtest/TestConfig/lib/testng.jar)
	at org.testng.TestNG.main(TestNG.java:1323) from jdk.internal.loader.ClassLoaders$AppClassLoader@ca632023(file:/home/jenkins/jenkins-agent/workspace/PullRequest-Sanity-JDK10-linux_ppc-64_cmprssptrs_le-OpenJ9/openj9/jvmtest/TestConfig/lib/testng.jar)
```

--- a/test/TestConfig/resources/excludes/latest_exclude_SE100.txt
+++ b/test/TestConfig/resources/excludes/latest_exclude_SE100.txt
@@ -23,7 +23,6 @@
 # Exclude tests temporarily
 
 org.openj9.test.vm.Test_MsgHelp:test_loadMessages_EN																	AN-https://github.ibm.com/runtimes/test/issues/46 generic-all
-org.openj9.test.java.lang.Test_Class:test_getModifiers_classTypes														1582 generic-all
 org.openj9.test.java.lang.management.TestRuntimeMXBean:testGetMBeanInfo													1626 generic-all
 org.openj9.test.java.lang.management.TestThreadMXBean:testGetMBeanInfo													1626 generic-all
 org.openj9.test.attachAPI.TestAttachAPI:test_agntld01																	238 generic-all

--- a/test/TestConfig/resources/excludes/latest_exclude_SE90.txt
+++ b/test/TestConfig/resources/excludes/latest_exclude_SE90.txt
@@ -23,7 +23,6 @@
 # Exclude tests temporarily
 
 org.openj9.test.vm.Test_MsgHelp:test_loadMessages_EN												                    AN-https://github.ibm.com/runtimes/test/issues/46 generic-all
-org.openj9.test.java.lang.Test_Class:test_getModifiers_classTypes														JTC-JAT-133182 generic-all
 org.openj9.test.java.lang.Test_ClassLoader																				134754 generic-all
 org.openj9.test.java.lang.Test_Thread:test_getContextClassLoader														125908 generic-all
 org.openj9.test.attachAPI.TestAttachAPI:test_agntld01																	238 generic-all

--- a/test/functional/Java8andUp/src_90/org/openj9/test/java/lang/Test_Class.java
+++ b/test/functional/Java8andUp/src_90/org/openj9/test/java/lang/Test_Class.java
@@ -1107,7 +1107,7 @@ public void test_getModifiers_classTypes() {
 		0x0009, // public static
 		0x0011, // public final
 		0x0019, // public final static
-		0x0008, // static
+		0x0000, //
 		0x0410, // static
 		0x0400, // abstract
 		0x1001, // public synthetic

